Story05 (#17)
alem-platform/bootcamp-js: story05/hybrid-creature/README.md

# The Hybrid Creature

| Expected file      |
| ------------------ |
| hybrid-creature.js |

### Instructions

> Write a function named `createHybridObject` that accepts an object and a method to add to the prototype of the object. The factory function should return the instance of the object that has been created.

Reviewer: ```suggestion
Write a function called `createHybridObject` that takes an object and a method to add to its prototype. The function should return the created object instance.
```

- The factory function `createHybridObject` should accept any parameters needed to create an object.
- The constructor function should define properties and methods for the objects being created.
- The factory function should return an instance of the object created by the constructor function.

> 

Reviewer: expected function
